### ISSUE TITLE
type of specimen

### ISSUE THREAD

**Issue by megbalk** [futres/template#6]
To better capture the type of specimen a measurement is on, we will have two fields: basisOfRecord (which will need to be adapted for zooarch data) and materialSampleID, which refers to the part of the organism is being sampled.
Additionally, we'll omit "aggregatedData" for now as we are not currently ingesting lots of specimens. #3

**Comment by jdeck88** [comment]
we have basisOfRecord and materialSampleID on the template at https://geome-db.org/workbench/template?projectId=156 ... we don't have aggregatedData.... so i think conditions met.

**Comment by megbalk** [comment]
Correct. We may need something in the ontology that is flexible for this term as specimen can mean entire individual organism, element of an individual, or an aggregation (which we aren't 
dealing with yet, but we are dealing with the former two cases)

**Comment by jdeck88** [comment]
Seems like this is linked to this issue in the FOVT repository? https://github.com/futres/fovt/issues/14 

Darwin core has this field: http://rs.tdwg.org/dwc/terms/organismScope

Definition | A description of the kind of Organism instance. Can be used to indicate whether the Organism instance represents a discrete organism or if it represents a particular type of aggregation.
-- | --
Comments | Recommended best practice is to use a controlled vocabulary. This term is not intended to be used to specify a type of taxon. To describe the kind of dwc:Organism using a URI object in RDF, use rdf:type (http://www.w3.org/1999/02/22-rdf-syntax-ns#type) instead.

Sounds like a plan would be develop a controlled vocabulary in FOVT and then use that vocabulary as available terms for organismScope
